File: 217-contains-duplicate.js


/**
 * @param {number[]} nums
 * @return {boolean}
 */
/**
 * @param {number[]} nums
 * @return {boolean}
 */
export function containsDuplicate(nums) {
  for (let i = 0; i < nums.length; i++) {
    const num = nums[i];
    for (let j = 0; j < nums.length; j++) {
      if (num === nums[j] && i !== j) {
        return true;
      }
    }
  }
  return false;
}

File: 217-contains-duplicate.py


/**
 * @param {number[]} nums
 * @return {boolean}
 */
# FILEPATH: /Users/jan/Projects/leet-problems/217-contains-duplicate.py
# BEGIN: ed8c6549bwf9
def containsDuplicate(nums):
  for i in range(len(nums)):
    num = nums[i]
    for j in range(len(nums)):
      if num == nums[j] and i != j:
        return True
  return False
# END: ed8c6549bwf9
